KthLastElement.java

```diff
@@ -0,0 +1,46 @@
+package com.katermar;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Created by USER on 5/15/2017.
+ */
+public class KthLastElement {

```

Comment:
No need to request new review. All changes done on this branch until it is merged will be part of a single review, so that there is a history of changes, comments etc.